rules.txt

```diff
@@ -0,0 +1,148 @@
+Panorama geral do projeto, vamos planejar a partir disso:
+
+## Unidades Espaciais ##
+- Fighters / Bombers
+  Podem dar jump a planetas proximos que tenham CS aliada
+  Diferenciados pelo uso de misseis ou torpedos
+
+- CapitalShips
+  Carregam Fighters/Bombers/Weapons/Troopers/Vehicles
+  Podem ter upgrades e/ou XP
+  Podem dar hyperjumps distantes
+
+- Transports
+  Carregam Fighters/Bombers/Troopers/Vehicles (Em menor qtde do q uma CS)
+  Podem dar hyperjumps planetas próximos
+
+- Shipyards
+  Constroem as naves em turnos
+  Tem "pontos de produção" para construir as unidades
+
+## Unidades Terrestres ##
+- HeadQuarter (NOVO)
+  Barrack quartel general do Squad
+  Só pode haver uma
+  Produz tropas e veículos
+  Abriga Heroes
+
+- Barracks (NOVO)
+  Produz tropas e veículos
+
+- Troopers
+  Soldados para ataques terrestres
+
+- Vehicles
+  Veiculos Terrestres. AT-AT, AT-ST etc
+
+## Unidades Especiais ## 
+- Heroes
+  Cumprem missões diversas. Espionagem, Sabotagem, Diplomacia etc
+
+## Produção das Shipyards e Barracks ##
+- Sua construção depende de créditos/metais e % de dominação do planeta
+-  Ideal poder construir baseado na sua capacidade e setando por exemplo: 20 X-Wings, 50% Calamari Cruiser, 30% Transports e ela ir produzindo sozinho. 
+-  Opção também para comprar unidades instantaneamente com créditos/metais
+
+## Movimentos ##
+   Tem q ser user friendly. Arrastar e soltar ele pede a qtde a ser movida, mostra o tempo de viagem em turnos baseado no algoritmo Dikstra.
+
+## Ataques ##
+- Caracterizados ao encontrar um esquadrão inimigo no planeta
+- Automático, com opção de formação de ataque, alvo primário, fuga depois de % perdida etc para variar o algoritmo de batalha. (dar opção extra para jogar XWA)
+
+## Mapa ##
+- Planetas variados dividos por setor tentando seguir o mapa oficial de SW
+- Rotas poderia ser esse esquema dos Hexágonos mesmo ou rotas pré definidas, como é hyperespaço uma CS não necessariamente tem q seguir a rota mas quando der um jump de 2 planetas ele contabilizada o tempo de viagem pela rota. A frota fica em "suspensão"
+- Mostrar os recursos do planeta, a % de dominação e quanto ele está produzindo para o squad
```

Comment:
% de dominacao parece over pra primeira versao, se tem naves os dois batalham e acabou, droparia inicialmente.

Reply:
Sim. Quando vc diz "dropas inicialmente" vc quer dizer enquanto a gente desenvolve e testa né? Quando for abrir pra geral acho q ja tem q ter.